wazuh/wazuh-kibana-app
# Added parenthesis to search bar query in PDF report

Hi team,

this PR fixes the PDF reports search bar query by encapsulating the query. It prevents issues when using queries with OR operators.

Closes #3853 

To test it: 

- Go to a dashboard with 2 o more agents
- Filter a query in the search bar with OR operators which would result in 1 agent per condition
like: `agent.name : *W* OR agent.name : *180*`
- Generate a PDF report and check the amount of agents in the table are the same as in the dashboard
